template.yaml

```diff
@@ -828,6 +828,29 @@ Resources:
             Method: get
             RestApiId: !Ref NvaPublicationApi
 
+  CreatePublicationFromImportCandidateFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: publication-rest
+      Handler: no.unit.nva.publication.create.CreatePublicationFromImportCandidateHandler::handleRequest
+      Runtime: java11
+      AutoPublishAlias: live
+      DeploymentPreference:
+        Type: AllAtOnce
+      Environment:
+        Variables:
+          ALLOWED_ORIGIN: '*'
+          IMPORT_CANDIDATES_TABLE: !Ref NvaImportCandidatesTable
+          TABLE_NAME: !Ref NvaResourcesTable
```

Comment:
Could TABLE_NAME be called NVA_RESOURCE_TABLE_NAME=

Reply:
When we instantiating ResourceService without passing in table name, it creates "default" ResourceService that operates in resource table. But i can rename it to make it clear yes, and pass this environment variable in to ResourceService constructor.